src/flows/context/flow.current.tsx

```diff
@@ -35,19 +45,76 @@ export const FlowProvider: FC = ({children}) => {
   const {flows, update, currentID} = useContext(FlowListContext)
   const [currentFlow, setCurrentFlow] = useState<Flow>()
 
+  const yDoc = useRef(new Y.Doc())
```

Comment:
should this be wrapped in a flag? Not sure how expensive Y.Doc() is

Reply:
Not sure you can wrap a hook in a flag

Reply:
That's fair